src/gm3/components/visibleLayers.js

```diff
@@ -60,7 +60,7 @@ class VisibleLayers extends React.Component {
 
         return (
             <Provider store={this.props.store}>
-                <div className="catalog visble-layers flat">
+                <div className="catalog visible-layers flat">
```

Comment:
Are we sure people aren't using the mis-spelled version in their custom CSS?  It worries me that this bug-fix might be a breaking change.  Should we include both class names for now?  Would that even work?

Reply:
It looks good other than that.

Reply:
We can leave both in there. Pushed a new commit.